File: javascript/src/models/httpPatchConfig.ts
// this file is @generated

export interface HttpPatchConfig {
  url?: string;
}

export const HttpPatchConfigSerializer = {
  _fromJsonObject(object: any): HttpPatchConfig {
    return {
      url: object["url"],
    };
  },

  _toJsonObject(self: HttpPatchConfig): any {
    return {
      url: self.url,
    };
  },
};

File: rust/src/models/http_patch_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct HttpPatchConfig {
    #[serde(skip_serializing_if = "Option::is_none")]
    pub url: Option<String>,
}

impl HttpPatchConfig {
    pub fn new() -> Self {
        Self { url: None }
    }
}

impl Default for HttpPatchConfig {
    fn default() -> Self {
        Self::new()
    }
}
